codewof/programming/content/en/merge-lists-odd-even/question.md

```diff
@@ -0,0 +1,5 @@
+# Merge lists
+
+Write a function `merge_lists(list_1, list_2)` that takes two lists containing integers and returns a third list that contains only odd numbers from the first list and even numbers from the second list.
+
+For example `merge_lists([11, 26, 17, 99, 42], [12, 43, 98, 36, 9, 13])` should return `[11, 17, 99, 12, 98, 36]`.
```

Comment:
The use of bold for print/return is just inconsistent; some questions it's on all of them; others it's for just the example; and others like this one are without bold at all